README.md

```diff
@@ -1,7 +1,17 @@
-# starter-kit
+# WICG Starter kit
 
 [![Greenkeeper badge](https://badges.greenkeeper.io/WICG/starter-kit.svg)](https://greenkeeper.io/)
-A simple starter kit for [WICG](https://wicg.io) specs.
+Welcome!  This is a simple starter kit for writing [WICG](https://wicg.io) specifications, and adding new awesome features to the web platform.  In this repo you'll find templates and guides for using them.
+
+## Process
+
+Start with an explainer. [Learn more](#TODO-slightlylate) and [get the template](https://github.com/WICG/starter-kit/blob/master/templates/explainer.md).  The explainer is critical to getting early engagement and understanding about what you'll be building.
+
+When you move into writing the spec itself, use a document framework such as [ReSpec](https://github.com/w3c/respec) or [Bikeshed](https://github.com/tabatkins/bikeshed).  This will ensure that you apply the appropriate metadata and provide details to help people locate the spec and understand the structure.
```

Comment:
It would be helpful if this doc outlined *how* to author god specs in both of these -- e.g. hygiene around spec references, how to do internal references correctly, pointers to IDL markup, etc. A skeleton spec in respec and bikeshed, demonstrating all this, would go along way.